src/EditorFeatures/CSharp/Highlighting/KeywordHighlighters/AsyncAwaitHighlighter.cs

```diff
@@ -108,10 +108,10 @@ private static bool TryAddAsyncOrAwaitKeyword(SyntaxToken mod, List<TextSpan> sp
                 if (spans.Count > 0)
                 {
                     var previousToken = mod.GetPreviousToken();
-                    var lastSpan = spans[spans.Count - 1];
+                    var lastSpan = spans[^1];
```

Comment:
i'm actually curious how this works.  `List<T>` doesn't have an indexer that takes System.Index...

Reply:
The compiled code doesn't use `Index` at all. The bytecode is the same before and after this change.